packages/components/src/Menu/Menu.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from "react";
+import uuid from "uuid";
+import { Button } from "../Button";
+import { Typography } from "../Typography";
+import { Icon, IconNames } from "../Icon";
+import styles from "./Menu.css";
+
+interface MenuProps {
+  /**
+   * Collection of action items.
+   */
+  readonly items: SectionProps[];
```

Comment:
I'm wondering if this should be an array or, if we should just use `children`…

Reply:
🤔 Children allows people to put non menu things inside the menu. What's your concern?